index.html

```diff
@@ -168,8 +168,7 @@ <h3>
 				</div>
 			</div>
 		</section>
-
-		<section id="about">
+        <section id="about">
```

Comment:
- [ ] Please make sure that your portfolio is fully responsive. While switching from mobile to desktop version, your `button` in `about section`  is hiding under the cards. See below for more information 🔽

## **_Your design 👇_**

https://user-images.githubusercontent.com/106119105/213690240-be0db69b-876c-4651-bb7a-d5507cda0a5a.mp4

## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213690202-2c99d00d-0882-410b-abb1-76249fa3b4a0.png)